src/components/autocomplete/components/asyncAutocomplete/AsyncAutocompleteContainer.tsx

```diff
@@ -388,8 +387,19 @@ export default function AsyncAutocompleteContainer({
 
           void contractingInsuranceFromTaxId(addUser, endpoint, params, value);
         } else {
-          const endpoint = addUser ? 'ONBOARDING_GET_INSTITUTIONS' : 'ONBOARDING_GET_PARTY_FROM_CF';
-          void handleSearchByTaxCode(addUser, endpoint, params, value);
+          const endpoint = addUser
+            ? 'ONBOARDING_GET_INSTITUTIONS'
+            : product?.id === 'prod-interop'
```

Comment:
By doing this, the search by tax code on infocamere will always be valid

Reply:
added secondary condition to search by tax code on infocamere only for a specific case